src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.model;
+
+public class User {

```

Comment:
We need to make `User` as entity class to work with it through Hibernate

Reply:
fixed